### DISCUSSION TITLE
Grafana Agent - Metrics failing to scrape?

### DISCUSSION THREAD

**Discussion by brngates98** [grafana-cold-storage/agent#5937] [Q&A] [answered]
I get this error when using a very basic grafana agent config for windows.

It uses the default windows integration agent config

Application | Grafana Agent            | level=warn msg="GET /integrations/windows_exporter/metrics 12.7530555s, error: write tcp 127.0.0.1:12345->127.0.0.1:50091: wsasend: An established connection was aborted by the software in your host machine. ws: false; Accept: application/openmetrics-text;version=1.0.0,application/openmetrics-text;version=0.0.1;q=0.75,text/plain;version=0.0.4;q=0.5,*/*;q=0.1; Accept-Encoding: gzip; User-Agent: GrafanaAgent/v0.38.0 (static; windows; binary); X-Prometheus-Scrape-Timeout-Seconds: 10; "

**Comment by Deborah-S** [comment]
I have the same warning message but haven't found a solution yet ...

**Reply by brngates98** [reply, marked as answer]
So i did eventually resolve this issue, i had to actually increase the time out settings for the default scrape timeout.
